### ISSUE TITLE
query mapper for single object using Observable<T> 

### ISSUE THREAD

**Issue by ziahaqi** [sockeqwe/sqlbrite-dao#35]
i try to get single user data using Observer<User> getUser(int id) method. but the subscriber dont get back enything, what wrong ? 
I've been using a query like this : 
        return query(SELECT(USER_PROJECTION)
                .FROM(User.TABLE_NAME)
                .WHERE(User.COL_ID + " = ?"))
                .args(String.valueOf(userId))
                .run()
                .mapToOne(new Func1<Cursor, User>() {
                    @Override
                    public User call(Cursor cursor) {
                        //....
                    }
                })
I have tried mapToOne(UserMapper.MAPPER). 
and the result still same.

**Comment by sockeqwe** [comment]
Do you subscribed to that observable? Is `Subscriber.onNext()` or `Subscriber.onError()` really never called?

**Comment by ziahaqi** [comment]
yes i did. I already subscribe method on my Presenter by this way :  

userObservable.observeOn(AndroidSchedulers.mainThread()).
                        subscribe(new Subscriber<User>() {
                            @Override
                            public void onCompleted() {
                                 Log.i(TAG, "onCompleted");
                            }

```
                        @Override
                        public void onError(Throwable e) {
                            mLoginView.showError(e.getMessage());
                        }

                        @Override
                        public void onNext(User user) {
                            mLoginView.onGetUserSuccess(user); 
                        }
                    });
```

**Comment by sockeqwe** [comment]
Can you provide me with some code that I can run and debug?

I really would be surprised if there is a bug in SQLBrite or SQLBrite-DAO
